modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -38,19 +39,105 @@ $ oc annotate Network.operator.openshift.io cluster \
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool master --type='merge' --patch \
+  '{ "spec": { "paused": true } }'
+----
++
+[source,terminal]
+----
+$ oc patch MachineConfigPool worker --type='merge' --patch \
+  '{ "spec":{ "paused" :true } }'
+----
+
+. Configure the OVN-Kubernetes cluster network provider with one of the following commands:
+
+.. To specify the network provider without changing the cluster network IP address block, enter the following command:
 +
 [source,terminal]
 ----
 $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
 ----
 
-. To confirm the migration disabled the OpenShift SDN default CNI network provider and removed all OpenShift SDN pods, enter the following command. It might take several moments for all the OpenShift SDN pods to stop.
+.. To specify a different cluster network IP address block, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{
+    "spec": {
+      "clusterNetwork": [
+        "cidr": "<cidr>",
+        "hostPrefix": "<prefix>"
+      ]
+      "networkType": "OVNKubernetes"
+    } 
+  }'
+----
++
+where `cidr` is a CIDR block and `prefix` is the slice of the CIDR block apportioned to each node in your cluster.
++
+IMPORTANT: You cannot change the service network address block during the migration.
+
+. Optional: You can customize the following settings for OVN-Kubernetes to meet your network infrastructure requirements:
++
+--
+* Maximum transmission unit (MTU)
+* Geneve (Generic Network Virtualization Encapsulation) overlay network port
+--
++
+To customize either or both of the previously noted settings, customize and enter the following command. If you do not need to change the default value, omit the key from the patch.
++
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type=merge \
+  --patch '{
+    "spec":{
+      "defaultNetwork":{
+        "ovnKubernetesConfig":{
+          "mtu":<mtu>,
+          "genevePort":<port>
+    }}}}'
+----
++
+where:
++
+--
+`mtu`::
+The MTU for the Geneve overlay network. This value is normally configured automatically, but if the nodes in your cluster do not all use the same MTU, then you must set this explicitly to `100` less than the smallest node MTU value.
+`port`::
+The UDP port for the Geneve overlay network.
+--
++
+.Example patch command 
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type=merge \
+  --patch '{
+    "spec":{
+      "defaultNetwork":{
+        "ovnKubernetesConfig":{
+          "mtu":1200
+    }}}}'
+----
+
+. Wait until the Multus daemon set pods restart. The name of the Multus pods is in form of `multus-<xxxxx>` where `<xxxxx>` is a random sequence of letters. It might take several moments for the pods to restart.
++
+[source,terminal]
+----
+$ watch oc get pod -n openshift-multus
```

Comment:
Suggest use this way to check the multus pods rollout.  'oc -n openshift-multus rollout status daemonset/multus'

Reply:
@huiran0826, what output does that command produce? Thanks!

Reply:
+ oc -n openshift-multus rollout status daemonset/multus
Waiting for daemon set "multus" rollout to finish: 1 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 1 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 1 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 2 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 2 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 2 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 2 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 3 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 3 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 3 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 3 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 4 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 4 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 4 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 4 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 5 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 5 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 5 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 5 out of 6 new pods have been updated...
Waiting for daemon set "multus" rollout to finish: 5 of 6 updated pods are available...
daemon set "multus" successfully rolled out